data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -37,6 +37,23 @@ MDNS_PUBLISHER_IMAGE=$(image_for mdns-publisher)
 HAPROXY_IMAGE=$(image_for haproxy-router)
 BAREMETAL_RUNTIMECFG_IMAGE=$(image_for baremetal-runtimecfg)
 
+# Copy machine-config-daemon binary from payload and run pivot
+if [ ! -f .pivot-done ]; then
+  mkdir /run/pivot /etc/pivot
+  touch /run/pivot/reboot-needed
+  echo "${MACHINE_CONFIG_OSCONTENT}" > /etc/pivot/image-pullspec
+
+  mkdir bin/
+  bootkube_podman_run \
+		--volume "$PWD/bin:/host/usr/local/bin:z" \
```

Comment:
This will relabel the host's `/usr/bin` - dangerous.  Let's instead do something like:
```
hostmcd=/usr/local/bin/machine-config-daemon
bootkube_podman_run --entrypoint=sh "${MACHINE_CONFIG_OPERATOR_IMAGE} cat /usr/bin/machine-config-daemon >${hostmcd}
chmod a+x ${hostmcd}
restorecon ${hostmcd}
```